Shaders/std/light.glsl

```diff
@@ -84,10 +93,10 @@ uniform sampler2D sltcMag;
 vec3 sampleLight(const vec3 p, const vec3 n, const vec3 v, const float dotNV, const vec3 lp, const vec3 lightCol,
 	const vec3 albedo, const float rough, const float spec, const vec3 f0
 	#ifdef _ShadowMap
-		, int index, float bias, bool receiveShadow
+		, int index, float bias, bool receiveShadow, bool transparent

```

Comment:
This will tell the shader not to render transparent shadows for transparent objects.